lib/container.js

```diff
@@ -42,7 +42,7 @@ class Container {
     this.createMocha();
     container.helpers = createHelpers(config.helpers || {});
     container.translation = loadTranslation(config.translation || null);
-    container.support = createSupportObjects(config.include || {});
+    container.support = createSupportObjects(config.include || {}, config.globalActionTimeout);
```

Comment:
I don't think we need to change how container works here. We need this param only for `actor` so let's not pass this var here